driver/registry_default_registration.go

```diff
@@ -8,8 +8,8 @@ import (
 	"github.com/ory/kratos/selfservice/flow/registration"
 )
 
-func (m *RegistryDefault) PostRegistrationPrePersistHooks(ctx context.Context, credentialsType identity.CredentialsType) (b []registration.PostHookPrePersistExecutor) {
-	for _, v := range m.getHooks(string(credentialsType), m.Config(ctx).SelfServiceFlowRegistrationAfterHooks(string(credentialsType))) {
+func (m *RegistryDefault) PostRegistrationPrePersistHooks(ctx context.Context) (b []registration.PostHookPrePersistExecutor) {
+	for _, v := range m.getHooks("", m.Config(ctx).SelfServiceFlowRegistrationBeforePersistHooks()) {
```

Comment:
Is there a reason for removing the credentials type? Before it would allow us to do different things depending on what credential type was used. Now, we can only run exactly one sequence of hooks for all credential types